### DISCUSSION TITLE
TuxGuitar

### DISCUSSION THREAD

**Discussion by juandspy** [CoderLine/alphaTab#845] [Q&A] [answered]
Are TuxGuitar files supported?

**Comment by Danielku15** [comment, marked as answer]
No, TuxGuitar files are not (yet?) supported and it is also not directly on the roadmap compared to other, more spread formats. 
But technically it should be not a problem to add it (with corresponding efforts). I think the format did not change since years. As TuxGuitar is open source, the format is known. 

If there would be a strong need, I could add it to the roadmap.

**Reply by juandspy** [reply]
Thanks for the answer @Danielku15 . I don't think it's that necessary as you can save TuxGuitar files as Guitar Pro 5, 6 and 7. I'll try to add a PR when I have time for this format.
